### PR TITLE
Add release notes

### PR DESCRIPTION
**Pull request by huayunh**
## Changes proposed in this Pull Request:

- Add release notes for our R26 release
-
-

## To Test:

- After `yarn && yarn start`, check:
  - the home page, where the release summary is
  - the detailed release notes
  - ~Note that packages marked with a `❗️` is waiting for master PR / publishing~

### CODE REVIEW

File: src/docs/release-notes/R26/FullNotes.mdx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+#### React
+
+-   ❗️@brightlayer-ui/react-components v6.2.0 ([Changelog](https://github.com/etn-ccis/blui-react-component-library/blob/master/CHANGELOG.md))
+-   @brightlayer-ui/react-themes v7.1.0 ([Changelog](https://github.com/etn-ccis/blui-react-themes/blob/master/CHANGELOG.md))
+-   @brightlayer-ui/react-auth-workflow v3.1.1 ([Changelog](https://github.com/etn-ccis/blui-react-workflows/blob/master/login-workflow/CHANGELOG.md))
+
+#### React Native
+
+-   ❗️@brightlayer-ui/react-native-components v7.1.0 ([Changelog](https://github.com/etn-ccis/blui-react-native-component-library/blob/master/CHANGELOG.md))
+-   ❗️@brightlayer-ui/react-native-auth-workflow v5.0.1 ([Changelog](https://github.com/etn-ccis/blui-react-native-workflows/blob/master/login-workflow/CHANGELOG.md))
```
Comment (JeffGreiner-eaton) [inline src/docs/release-notes/R26/FullNotes.mdx:51]: The RN auth workflow will be v5.1.0

File: src/docs/release-notes/R26/FullNotes.mdx [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+## GitHub Enterprise Migration
+
+-   Migrated our code base to a [new location](https://github.com/etn-ccis?q=blui), organized using GitHub Enterprise.
```
Comment (joebochill) [inline src/docs/release-notes/R26/FullNotes.mdx:5]:
```suggestion
-   Migrated our code base to a [GitHub Enterprise](https://github.com/etn-ccis?q=blui) to align with other Eaton platform teams.
```

File: src/docs/release-notes/R26/FullNotes.mdx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+-   Added new text field-based components, such as Physical Location Address Text Fields, Phone Number Text Fields, to the Figma component sticker sheet.
+-   Added a new component, App Logo Template, to help designers design their product app store logos.
+-   Added iPhone's "Dynamic Island" as a component.
+-   Added a Breadcrumb component.
+
+## Angular
+
+-   Updated our packages to support Angular 14.
+
+## React
```
Comment (joebochill) [inline src/docs/release-notes/R26/FullNotes.mdx:25]: This was a bugfix, I would either leave it out or lump it under the bugs heading. It's a very low level fix to be highlighted in release notes.

File: src/docs/release-notes/R26/FullNotes.mdx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+-   Removed our packages' dependency on `@mui/styles`.
+
+## Misc
+
+-   Bug fixes & minor updates.
+-   The Brightlayer UI CLI can now only generate new React and React Native projects in typescript.
```
Comment (joebochill) [inline src/docs/release-notes/R26/FullNotes.mdx:32]:
```suggestion
-   Dropped support for Javascript — moving forward all of our packages, examples, and templates will exclusively use Typescript. The Brightlayer UI CLI will now only generate typescript projects.
```

File: src/docs/release-notes/R26/FullNotes.mdx [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+#### Angular
+
+-   @brightlayer-ui/angular-components v8.0.1 ([Changelog](https://github.com/etn-ccis/blui-angular-component-library/blob/master/CHANGELOG.md))
+-   @brightlayer-ui/angular-themes v8.0.0 ([Changelog](https://github.com/etn-ccis/blui-angular-themes/blob/master/CHANGELOG.md))
+-   @brightlayer-ui/angular-auth-workflow v4.0.0 ([Changelog](https://github.com/etn-ccis/blui-angular-workflows/blob/master/login-workflow/CHANGELOG.md))
+
+#### React
+
+-   ❗️@brightlayer-ui/react-components v6.2.0 ([Changelog](https://github.com/etn-ccis/blui-react-component-library/blob/master/CHANGELOG.md))
```
Comment (joebochill) [inline src/docs/release-notes/R26/FullNotes.mdx:44]: Why do these have exclamation points?

Comment (huayunh) [reply src/docs/release-notes/R26/FullNotes.mdx:44]: From the PR description: 

>   - Note that packages marked with a `❗️` is waiting for master PR / publishing

**Comment by JeffGreiner-eaton** [comment]
are the links here correct for 

Added style guidelines for designing [app store logos](https://github.com/etn-ccis/blui-doc-it/blob/feature/r26-release-notes/style/logo).
Added design guidelines for the [buttons](https://github.com/etn-ccis/blui-doc-it/blob/feature/r26-release-notes/patterns/buttons) design pattern.
Added guidelines for how to request, conduct and interpret a [UI design audit](https://github.com/etn-ccis/blui-doc-it/blob/feature/r26-release-notes/design/design-audit).

**Comment by huayunh** [comment]
@JeffGreiner-eaton Sorry, is that some issue you noticed with this PR? Relative link works in the firebase links, but not when you try to preview them directly in GitHub source code

**Review by JeffGreiner-eaton — APPROVED** [review]

**Pull request merged**